.github/workflows/benchmark.yml

```diff
@@ -9,19 +9,17 @@ on:
 jobs:
   benchmark:
     runs-on: ubuntu-latest
-    env:
-      BASE: ${{ github.event.pull_request.base.ref }}
-      HEAD: ${{ github.sha }}
 
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
-      # fetch the target branch from origin too (just the head)
-      # and check it out into a local branch
-      - name: Checkout PR and target branch
-        run: |
-          git fetch --no-tags --depth=1 origin $BASE
+      - run: |
```

Comment:
How come this one doesn't have a `name`?

Reply:
They don't need names, but I've added one now anyway